modules/core/src/main/java/org/apache/ignite/cache/query/SqlFieldsQuery.java

```diff
@@ -360,18 +363,19 @@ public boolean isReplicatedOnly() {
     /**
      * Sets lazy query execution flag.
      * <p>
-     * By default Ignite attempts to fetch the whole query result set to memory and send it to the client. For small
+     * If {@code lazy=false} Ignite will attempt to fetch the whole query result set to memory and send it to the client. For small
      * and medium result sets this provides optimal performance and minimize duration of internal database locks, thus
      * increasing concurrency.
      * <p>
      * If result set is too big to fit in available memory this could lead to excessive GC pauses and even
-     * OutOfMemoryError. Use this flag as a hint for Ignite to fetch result set lazily, thus minimizing memory
+     * {@link OutOfMemoryError}. Use this flag as a hint for Ignite to fetch result set lazily, thus minimizing memory
      * consumption at the cost of moderate performance hit.
      * <p>
-     * Defaults to {@code false}, meaning that the whole result set is fetched to memory eagerly.
+     * Defaults to {@code true}, meaning that the only first page of result set is fetched to memory.
      *
      * @param lazy Lazy query execution flag.
      * @return {@code this} For chaining.
+     * @deprecated Use {@link #setPageSize(int)} instead.
```

Comment:
why you deprecate these methods? How I can use `setPageSize` instead of these methods?